src/graphql/account.rs

```diff
@@ -286,9 +286,24 @@ impl AccountMutation {
     ) -> Result<AuthPayload> {
         let store = crate::graphql::get_store(ctx).await?;
         let account_map = store.account_map();
+        let client_ip = get_client_ip(ctx);
 
         if let Some(mut account) = account_map.get(&username)? {
             if account.verify_password(&password) {
+                if let Some(allow_access_from) = account.allow_access_from.as_ref() {
```

Comment:
From a security perspective, I think it would be better to check the IP address *before* verifying the password. This way, we prevent potential brute-force attacks from unauthorized IP addresses. Currently, the check is done after password verification, which could lead to unnecessary password checks. Would it be possible to reorder these checks for added security?